test/dotnet-run.Tests/GivenDotnetRunRunsCsProj.cs

```diff
@@ -645,6 +646,40 @@ public void ItRunsWithTheSpecifiedVerbosity()
             }
         }
 
+        [Fact]
+        public void ItDoesNotShowImportantMessageLevelByDefault()
```

Comment:
I think this would read better:

```suggestion
        public void ItDoesNotShowImportantLevelMessageByDefault()
```

Reply:
used suggested